build.sbt

```diff
@@ -62,7 +64,8 @@ val baseSettings = Seq(
   },
   scalacOptions in (Compile, console) += "-Yrepl-class-based",
   fork in Test := true,
-  javaOptions in ThisBuild ++= Seq("-Xss2048K")
+  javaOptions in ThisBuild ++= Seq("-Xss2048K"),
+  addCompilerPlugin("org.spire-math" % "kind-projector" % "0.9.8" cross CrossVersion.binary)
```

Comment:
What's this for?

Reply:
To simplify lambda types. It's required for sane use of type classes like `Functor[F[_]]` for things like `F[_, _]`

Reply:
Example:
https://github.com/finagle/finch/blob/219c0a147ac27dc2c3f59ce4a9cf3ceb97902519/json-test/src/main/scala/io/finch/test/JsonLaws.scala#L45